libs/telemetry/events/bundle_init.go

```diff
@@ -0,0 +1,23 @@
+package events
+
+// Corresponds to the `DatabricksCliBundleInitEvent` proto message in `databricks_cli_log.proto`
+// as of 20 Dec 2024.
+type BundleInitEvent struct {
+	// UUID associated with the DAB itself. This is serialized into the DAB
+	// when a user runs `databricks bundle init` and all subsequent deployments of
+	// that DAB can then be associated with this init event.
+	Uuid string `json:"uuid,omitempty"`
```

Comment:
Can it actually be empty?

Reply:
We'll always set it client side. It could probably be empty as far as server-side validation is concerned.